### DISCUSSION TITLE
Embedding Grafana into my own webpage

### DISCUSSION THREAD

**Discussion by dkerr64** [jasonacox/Powerwall-Dashboard#220] [Show and tell]
Let me just start by saying that this is totally awesome.  It's amazing what I can do with this tool.  For example, I have a number of IP cameras and I build a webpage that shows them.  All but one of these cameras provide a 4x3 aspect ratio image.  The most recent camera I bought provides a 16x9 aspect ratio.  When I add that into the grid of 4x3's I have some blank space to fill.  I've been wanting to add some useful home stats to that space.  Today I have...

<img width="1398" alt="image" src="https://user-images.githubusercontent.com/17837612/226496844-6becd59f-d181-4c3b-ad1b-90d47bad177c.png">

And that is totally awesome.

To make this really fly, I need https://github.com/jasonacox/pypowerwall/issues/31 and #219 and #130.  I'd be happy to help with any of these if I can.

David

**Comment by jasonacox** [comment]
Beautiful dashboard @dkerr64 !

**Reply by jasonacox** [reply]
Please see updates to #219 and #130.  Also, let me know if you want to try your hand at a pull request for https://github.com/jasonacox/pypowerwall/issues/31 😉

**Reply by dkerr64** [reply]
PR sent for pypowerwall and I have updated my docker image with the settings you recommended for #219 and #130 which look to be working great.

Thanks
David
